src/core/common/trickle_timer.cpp

```diff
@@ -53,6 +53,61 @@ TrickleTimer::TrickleTimer(Instance &aInstance, Handler aHandler)
 {
 }
 
+void TrickleTimer::GetLastTimerStart(TimeMilli &aStartTime)
```

Comment:
Can simply return `TimeMilli`. We do this in a bunch of other places, for example:

https://github.com/openthread/openthread/blob/874b6beca8c0ce09f2d296652d234c0d011ad1d1/src/core/thread/network_data_leader_ftd.hpp#L253